[ONNX] Bump up ONNX submodule to a82c6a7010e2e332d8f74ad5b0c726fd47c85376

houseroad opened the pull request:
Summary: we only bump the submodule in oss to unblock some works

Test Plan: ci

Differential Revision: D21830800

KsenijaS commented:
following tests need to be disabled in caffe2/python/onnx/tests/onnx_backend_test.py:
test_dropout_default
test_.*adam

tests that are renamed:
test_softmax_cross_entropy.* -> test_sce.*
test_negative_log_likelihood.* -> test_nllloss.* 

Ops inverse and mean square error are removed from ONNX so we don't need following tests in 
caffe2/python/onnx/tests/onnx_backend_test.py:
test_inverse.*
test_mean_square_distance_.*


Otherwise it looks good. Thanks!

houseroad commented:
@KsenijaS do you mind to help me find all other tests I need to skip? 

Right now, I am doing so, it's quite annoying :-)

KsenijaS commented:
@houseroad Sure, I already tested it and you can find it here: https://github.com/pytorch/pytorch/pull/39266/files

KsenijaS commented:
@houseroad Also following tests have been recently added to ONNX:
onnx/backend/test/data/node/test_cast_BFLOAT16_to_FLOAT
onnx/backend/test/data/node/test_cast_FLOAT_to_BFLOAT16

Not sure if they are already skipped in caffe2 backend tests

spandantiwari reviewed (approved):
Thank you upgrading the ONNX version.

houseroad committed:
[ONNX] Bump up ONNX submodule to a82c6a7010e2e332d8f74ad5b0c726fd47c85376 (#39372)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/39372

we only bump the submodule in oss to unblock some works

Test Plan: ci

Reviewed By: hl475

Differential Revision: D21830800

fbshipit-source-id: 53c1307577dcf53a2cc5a908fbc9fb273053a9f1
